_episodes/07-errors.md

```diff
@@ -70,16 +66,16 @@ You can determine the number of levels by looking for the number of arrows on th
 In this case:
 
 1.  The first shows code from the cell above,
-    with an arrow pointing to Line 8 (which is `favorite_ice_cream()`).
+    with an arrow pointing to Line 9 (which is `lucky_dice(3)`).
```

Comment:
Line 1 ?

Reply:
1 as in the first level of the traceback.

Reply:
my bad! `Line 9` is correct